README.md

```diff
@@ -14,10 +14,3 @@ To use:
 3. In iTunes Connect, create and configure any IAP products for test.
 4. Enumerate the corresponding IAP identifiers in the ProductIdentifiers.plist.
 5. Build and run.
-
-
-Licence
```

Comment:
It's debatable, I suppose. But the generally accepted convention for some time now is to specify the license in a file called "LICENSE" at the root of a project. Some projects do also point out the license in the README, but it just means there are multiple references that must be maintained if the license is ever changed (e.g. to Apache License 2.0). And which one is canonical? Keep it DRY, right? ;)